packages/salesforcedx-vscode-core/package.json

```diff
@@ -364,9 +352,49 @@
           "command": "sfdx.force.source.delete.current.file",
           "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest' && editorIsOpen"
         },
+        {
+          "command": "sfdx.force.set.default.org",
```

Comment:
This command was always displayed as part of the command palette because we forgot to add it here.